doc/faq.rst

```diff
@@ -3,6 +3,25 @@ FAQ
 
 This is the list of Frequently Asked Questions for restic.
 
+Can restic resume an interrupted backup?
+----------------------------------------
+
+Yes it can. When backing up, restic periodically writes index files to keep a record of the uploaded data.
```

Comment:
Not sure if this was there when I left my previous comment, but I'd rephrase "Yes it can" by something like "It does by default:" (note colon). This is the first bullet point in the FAQ (deservedly so), and a three-word answer comes over as a little snug.

Reply:
I didn't fully agree with your suggestion, but I changed the title from "Can" to "Will" and changes the first sentence.